canonical-web-and-design/charmhub.io
# Build tab for docs

## Done

- Remove the details tabs that are not relevant
- Create separate page for each details tab
- Refactor the templates for details tabs

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045/wordpress/docs
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See all tabs work properly


## Issue / Card

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1621

## Screenshots

[if relevant, include a screenshot]
